src/status_im/hardwallet/core.cljs

```diff
@@ -62,7 +62,8 @@
     (if navigate-to-browser?
       (fx/merge cofx
                 {:db (assoc-in db [:hardwallet :on-card-connected] nil)}
-                (wallet/discard-transaction)
+                ;;TODO use new signing flow
+                ;;(wallet/discard-transaction)
```

Comment:
Could you change it to `signing/discard`  as it seems like doing what `wallet/discard-transaction` did before

Reply:
Or maybe put it into comments for later